app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationBottomSheet.kt

```diff
@@ -284,6 +284,7 @@ fun GroupHeader(title: String, groupColorValue: Long) {
 
 @Composable
 fun buildItems(items: List<@Composable () -> Unit>) {
+    Divider()
```

Comment:
Is this divider without any parameters necessary here or did it get here by accident? 😄